packages/flutter/lib/src/gestures/drag_details.dart

```diff
@@ -185,21 +192,36 @@ class DragUpdateDetails {
   ///    coordinate space of the event receiver.
   final Offset globalPosition;
 
+  /// The kind of the device that initiated the event.
+  final PointerDeviceKind? kind;
+
   /// The local position in the coordinate system of the event receiver at
   /// which the pointer contacted the screen.
   ///
   /// Defaults to [globalPosition] if not specified in the constructor.
   final Offset localPosition;
 
+  /// A delta offset from the point where the drag initially contacted
+  /// the screen to the point where the pointer is currently located (the
+  /// present [globalPosition]) when this callback is triggered.
+  final Offset offsetFromOrigin;
```

Comment:
Which origin should this be if this is a double tap and drag? may be good to document this

Reply:
It would be the global position of the most recent tap down, i.e. the second tap down of a double tap and drag.